### PR TITLE
Adjust to new JP skill locations

### LINKED ISSUE #796: Append Skills from JP breaks the support picker
Before opening an issue:
- [x] I tested the latest release
- [x] I looked at other issues (even the closed ones)
- [x] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)
- [x] I'll fill this template properly

**Describe the bug**
Support picker no longer working with the Append Skills released for JP's 6th anniversary

**Screenshots** / **Video**
https://imgur.com/a/CSdZv4P

**Device Info (please fill at-least 'Device model'):**
 - Device model: BlueStacks Hyper-V 4.280
 - Screen size: 1920x1080
 - Android Version: Android 7.1.1
 - RAM: 4GB

### PR DESCRIPTION
**Pull request by reconman**
Fixes #796

### CODE REVIEW

File: scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/modules/Support.kt [hunk 1]
```diff
@@ -377,10 +375,16 @@ class Support(
         val y = bounds.y + 325
         val x = bounds.x + 1627
 
+        val skillMargin = when(prefs.gameServer) {
+            GameServerEnum.Jp -> 90
+            else -> 155
+        }
+        prefs.gameServer == GameServerEnum.Jp
```
Comment (MathewSachin) [inline scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/modules/Support.kt:382]: I believe this line wasn't intended.

Comment (reconman) [reply scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/modules/Support.kt:382]: Right

**Review by MathewSachin — APPROVED** [review]

**Comment by MathewSachin** [comment]
The only problem is if the user switches the skill display with a button they added next to the support refresh button.

**Comment by MathewSachin** [comment]
Why do you force push everytime? 🤔 
Normal pushes should be fine since we squash merge anyways.

**Comment by reconman** [comment]
> Why do you force push everytime? 🤔
> Normal pushes should be fine since we squash merge anyways.

Used to from work where for some reason people prefer force pushing over squashing, I don't know why.

And when juniors don't amend/squash their commits themselves, 20 "code review correction" commits get merged into master.

**Pull request merged**